### ISSUE TITLE
Name: Dockter, Docktor, Dokta, Tudock, Dave?

### ISSUE THREAD

**Issue by nokome** [stencila/dockter#37]
The current name of this tool is Dockter. It's a portmanteau of doctor (either the medical or PhD kind) and Docker (the container platform) because it's aim is to make it easier for researchers (either the medical or PhD kind :slightly_smiling_face: ) to create Docker containers.

But there are five issues with the name:

- :speaking_head: Hard to distinguish in verbal communications: "Was that Docker or Dockter?"
- :writing_hand: Ambiguity in spelling: "Dockter" or "Docktor"?
- :desktop_computer:  Confusion at the command line: `dockter build .` is very similar `docker build .` and looking for that extra `t` can consume an extra brain cycle or two to figure out which tool you are using.
- :computer: Autocomplete at the command line: many people use autocomplete in the shell and having the first four letters the same makes this less useful.

As the famous joke says (well actually a riff on the original joke):

> There are 2 hard problems in computer science: cache invalidation, naming things, and off-by-1 errors.
[Leon Bambrick](https://twitter.com/secretGeek/status/7269997868)

Please add your thoughts and suggestions in comments below :pray:. If you like someone's suggested name please :+1: it

**Comment by giorgiosironi** [comment]
I agree a problem with `dockter` is not being specific enough to be unique in searches:
![907ce6ad-84e8-45a0-a730-f280a40f9735](https://user-images.githubusercontent.com/160299/47708722-95821b80-dc26-11e8-834c-af67e7fce1d1.png)
It also appears to be in use in [some other projects](https://github.com/search?q=dockter) as a pun.

**Comment by beneboy** [comment]
I like `Dokta`. It seems to be in use in the musical world but not in software.

**Comment by apawlik** [comment]
+1 for `Dokta` from me too.

**Comment by nokome** [comment]
Pinging @finlay for his vote :)

**Comment by finlay** [comment]
dokta works for me too. lower case though, right ?

**Comment by nokome** [comment]
Yep `dokta` at the command line

**Comment by TimVanMourik** [comment]
Taking a [tweet](
https://twitter.com/stencila/status/1058611711693864960) a bit too seriously here 😎 
My two suggestions are: “[Docker Who](https://twitter.com/tim_van_mourik/status/1058264465970470912)” or, as I’m a big fan of recursive animal acronyms, something like “Dodo: DOdo makes DOcker images”!

**Comment by nokome** [comment]
Thanks for the suggests @TimVanMourik ! My concerns though are:

- `dockerwho` (funny as it is) is quite long to have to type at the command line :space_invader: 
- `dodo` is nice, but also implies this tool might go extinct soon :slightly_smiling_face:

**Comment by TimVanMourik** [comment]
* Haha, without necessarily advocating `dockerwho`, my general rule is: "No programmer has ever spent more time on typing than on thinking about what to type," so I tend to advocate expressive variable names. And then anything below 10 chars is not too long, IMO, especially with autocomplete. But I totally understand that it is a bit too funny and not quite conveying the professional impression that you'd like. 
* And sure, if you don't wanna jinx your tool with calling it `dodo`, by all means.

Suggestion: If the coin lands on `dokta`, give the function a 'who' argument that is an alias of 'help' 😎

**Comment by nokome** [comment]
Oooh, I _really_ like the `who` argument suggestion (@TimVanMourik, you should be a CLUX (command line user experience ) designer!)

But how about we leverage the fact that we generate a JSON-LD document with all the software requirements for a project...**including their authors**. The `who` subcommand could be a "standing on the shoulders of giants" list of the authors of the current project (if available) and all of the software packages it relies on. e.g.

```bash
$ dockter who
Roger Bivand (rgdal, sp), Tim Keitt (rgdal), Barry Rowlingson (rgdal), Edzer Pebesma (sp)
```

Which for reference is based on filtering the `.environ.jsonld` file for one of our test fixtures

```bash
jq '.softwareRequirements[0].softwareRequirements[] | .author[] | .name' tests/fixtures/r-spatial/.environ.jsonld
```

**Comment by apawlik** [comment]
Many thanks to everyone for your comments and feedback. And the winner is:
**dockter** !
I am closing this issue for now but thanks to @TimVanMourik we are planning to add `dockter who` command (see #55)
